### PR TITLE
Add QOS and buffer profiles for SKU Arista-7800R3-48CQM2-C48

### PR DESCRIPTION
**Pull request by ysmanman**
#### Why I did it

QOS and buffer profiles are missing for Arista SKU Arista-7800R3-48CQM2-C48.

#### How I did it

Created a QOS and buffer profiles for Arista-7800R3-48CQM2-C48. The profiles are the same of those for Arista-7800R3-48CQ2-C48 because the forwarding ASICs are the same generation in both SKUs.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Comment by arlakshm** [comment]
Hi @ysmanman, Thanks for adding this. Can you please add unit test for this SKU as well, like the one added for` arista7800r3-48cq2-lc`

**Comment by ysmanman** [comment]
> Hi @ysmanman, Thanks for adding this. Can you please add unit test for this SKU as well, like the one added for` arista7800r3-48cq2-lc`

Hi @arlakshm, I added unit test for the sku. Please take a look.

**Comment by neethajohn** [comment]
@ysmanman, please update the PR description

File: device/arista/x86_64-arista_7800r3_48cqm2_lc/Arista-7800R3-48CQM2-C48/qos.json.j2 [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{%- macro generate_wred_profiles() %}
+    "WRED_PROFILE": {
+        "AZURE_LOSSLESS" : {
+            "wred_green_enable"      : "true",
+            "wred_yellow_enable"     : "true",
+            "wred_red_enable"        : "true",
+            "ecn"                    : "ecn_all",
+            "green_max_threshold"    : "2097152",
+            "green_min_threshold"    : "250000",
```
[the remark this answers is not in the document]
Comment (ysmanman) [reply device/arista/x86_64-arista_7800r3_48cqm2_lc/Arista-7800R3-48CQM2-C48/qos.json.j2:9]: @neethajohn According to our discussion with BRCM,  WRED profile on non-DNX should work on DNX too. These values were coming from a Arista non-DNX SKU and BRCM thought they are reasonable.

**Comment by ysmanman** [comment]
> @ysmanman, please update the PR description

@neethajohn Updated PR description.

File: device/arista/x86_64-arista_7800r3_48cqm2_lc/Arista-7800R3-48CQM2-C48/qos.json.j2 (hunk 1 above)
[the remark this answers is not in the document]
Comment (neethajohn) [reply device/arista/x86_64-arista_7800r3_48cqm2_lc/Arista-7800R3-48CQM2-C48/qos.json.j2:9]: Thanks for the update. We will go with this setting for now. We can always change them later if needed.

**Review by neethajohn — APPROVED** [review]

**Review by arlakshm — APPROVED** [review]

**Pull request merged**